### DISCUSSION TITLE
Custom errors with error codes

### DISCUSSION THREAD

**Discussion by StefanSelfTaught** [nextauthjs/next-auth#1254] [Help] [answered]
I saw in the docs that you can add a custom page for authentication errors and an error code is passed in a query string as `?error=` . Is there any list with all of this error codes so I can make use of them?

### From the docs:
![image](https://user-images.githubusercontent.com/43994418/106901487-7f663e00-6700-11eb-8c92-b33feb115693.png)

**Comment by balazsorban44** [comment, marked as answer]
not a list per se, but you can see the cases for errors here, lowercased:

https://www.github.com/nextauthjs/next-auth/tree/canary/src%2Fserver%2Fpages%2Ferror.js

**Comment by StefanSelfTaught** [comment]
Thanks! Should I add those in the docs?

**Reply by balazsorban44** [reply]
yes, thanks! if you find a good place for it, I think that would be useful!
